### PR TITLE
OSDOCS-4083: Added OVN-Kubernetes as the default networking plug-in

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-4083

Link to docs preview:
https://53625--docspreview.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/planning-migration-3-4.html#migration-preparing-networking

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@jboxman-rh @abhat Can you please review? I linked to the section on migrating from OpenShift SDN, but let me know if that section is not actually relevant for when coming from 3.11 and if I should just link to a generic "About" section instead, like: https://docs.openshift.com/container-platform/4.12/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html#about-ovn-kubernetes

Preview: https://53625--docspreview.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/planning-migration-3-4.html#migration-preparing-networking

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc [hunk 1]
```diff
@@ -148,6 +148,13 @@ If your {product-title} 3.11 cluster used the `ovs-subnet` or `ovs-multitenant`
 
 For more information, see xref:../networking/network_policy/about-network-policy.adoc#about-network-policy[About network policy].
 
+[discrete]
+==== OVN-Kubernetes as the default networking plug-in
+
```
Comment (ahardin-rh) [inline migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:153]: OVN-Kubernetes as the default networking plug-in in Red Hat OpenShift Networking (per a discussion with Marc Curry around new branding)

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc [hunk 2]
```diff
@@ -148,6 +148,13 @@ If your {product-title} 3.11 cluster used the `ovs-subnet` or `ovs-multitenant`
...
+The default networking plug-in for {product-title} 3.11 was OpenShift SDN. In {product-title} {product-version}, OVN-Kubernetes is now the default networking plug-in.
```
Comment (ahardin-rh) [inline migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:154]: In {product-title} 3.11, OpenShift SDN was the the default networking plug-in in Red Hat OpenShift Networking.

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc (hunk 1 above)
Comment (bergerhoffer) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:153]: Thanks @ahardin-rh updated throughout!

**Comment by stevsmit** [comment]
lgtm

**Comment by bergerhoffer** [comment]
@jboxman-rh Can you recommend someone to QE review this?

**Comment by bergerhoffer** [comment]
Hi @anuragthehatter - can you please QE review this? (or let me know if you're not the right QE for this :slightly_smiling_face: ) Thanks!

Preview: https://53625--docspreview.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/planning-migration-3-4.html#migration-preparing-networking

**Comment by mffiedler** [comment]
@anuragthehatter @weliang1 PTAL

**Pull request merged**